### ISSUE TITLE
Highlight to copy - SSH

### ISSUE THREAD

**Issue by farhank3389** [FreeRDP/Remmina#1434]
Is there a way to enable highlight to copy and paste to client? I can highlight to copy and paste on the remote machine but I can't paste into the client. Seems like it is not correctly copying to the clipboard?

* Client: Fedora 27
* Remmina version: 1.2.0-rcgit-20 (git rcgit-20)
* Desktop environment: Cinnamon 
* Connecting to: Centos 7 
* Connecting via: SSH 

Thanks

**Comment by antenore** [comment]
Use the copr package to upgrade Remmina has explained in the wiki.
You are using an old version of Remmina

**Comment by farhank3389** [comment]
I am using Fedora 27 but the latest https://copr-be.cloud.fedoraproject.org/results/hubbitus/remmina-next/ only goes up to Fedora 25. @antenore

**Comment by antenore** [comment]
Take a look here https://bodhi.fedoraproject.org/updates/FEDORA-2018-5730340aa9 
@hubbitus has posted a package for testing

**Comment by farhank3389** [comment]
I installed the version from https://bodhi.fedoraproject.org/updates/FEDORA-2018-5730340aa9 but I still have the same issue

**Comment by antenore** [comment]
I don't get what is the problem, sorry.

Can you rephrase?

**Comment by farhank3389** [comment]
If I highlight some text in an SSH session, the data gets copied to the clipboard and I can paste between Remmina SSH sessions. The problem is that I can't paste the same data to a local terminal or anything else on my fedora machine. The data that gets pasted is not what I highlighted in Remmina.

E.g If I copy some text that says "fedora" from a local terminal onto the clipboard, I can paste to Remmina. If I then copy some text that says "SSH" from a Remmina SSH session, I can paste "SSH" to other SSH sessions but if I paste to a local terminal or notepad, "fedora" will get pasted and not "SSH".

Apologies I wasn't clear earlier

**Comment by antenore** [comment]
No problem, thanks

**Comment by antenore** [comment]
It works for me, so it must be a clipboard issue in general.

I use gpaste as clipboard manager, I don't know how to help you.

If you use wayland, can you try to install XWayland and test with xclip (check the man page because you have 3 clipboard available)

**Comment by farhank3389** [comment]
Ah okay, thanks for your help

**Comment by antenore** [comment]
If you find anything else don't hesitate to get back to me
